backend/wmg/data/load_cube.py

```diff
@@ -35,28 +35,32 @@ def upload_artifacts_to_s3(
     information about the data schema version, the snapshot id, and the location
     of the snapshot.
 
-    TODO(prathap): This part of the docstring will be modified when work starts on ticket:
-    https://github.com/chanzuckerberg/single-cell-data-portal/issues/5166
+    The destination_path is determined by the following logic:
+    - If `snapshot_schema_version` is set, then the destination path is
+        s3://<environment_bucket_name>/snapshots/<data_schema_version>/
+    - Otherwise, the destination path is the root directory: s3://<environment_bucket_name>/
```

Comment:
I don't believe there is any `if-else` logic here. There is no option to omit `snapshot_schema_version`. I think this comment should just read:

```
The <destination_path> for the snapshot is `s3://<environment_bucket_name>/snapshots/<data_schema_version>/`